tests/test_slurmclusterexecutor.py

```
@@ -0,0 +1,90 @@
+            cloudpickle_register(ind=1)
+            fs1 = exe.submit(mpi_funct, 1)
+            self.assertFalse(fs1.done())
+            self.assertEqual(fs1.result(), [(1, 2, 0), (1, 2, 1)])
+            self.assertTrue(fs1.done())
+            self.assertEqual(len(os.listdir("executorlib_cache")), 4)
+            for file_name in os.listdir("executorlib_cache"):
+                file_path = os.path.join("executorlib_cache", file_name )
+                os.remove(file_path)
+                if ".h5" in file_path:
+                    task_key = file_path[:-5] + "_i.h5"
+                    dump(file_name=task_key, data_dict={"a": 1})
```

Ruff (0.11.9)

Undefined name `dump`

(F821)